docs/resources/database.md

```diff
@@ -57,7 +57,7 @@ resource "snowflake_database" "from_share" {
 - `data_retention_time_in_days` (Number) Number of days for which Snowflake retains historical data for performing Time Travel actions (SELECT, CLONE, UNDROP) on the object. A value of 0 effectively disables Time Travel for the specified database. Default value for this field is set to -1, which is a fallback to use Snowflake default. For more information, see [Understanding & Using Time Travel](https://docs.snowflake.com/en/user-guide/data-time-travel).
 - `from_database` (String) Specify a database to create a clone from.
 - `from_replica` (String) Specify a fully-qualified path to a database to create a replica from. A fully qualified path follows the format of `"<organization_name>"."<account_name>"."<db_name>"`. An example would be: `"myorg1"."account1"."db1"`
-- `from_share` (Map of String) Specify a provider and a share in this map to create a database from a share.
+- `from_share` (Map of String) Specify a `provider` and a `share` in this map to create a database from a share. As of version 0.87.0, the `provider` field requires the format of `"<account_locator>"`, which is now case-sensitive.
```

Comment:
Hey @jacobcbeaudin, this part is also generated based on https://github.com/Snowflake-Labs/terraform-provider-snowflake/blob/88944e7a02b1ff930e1a39bc519780c124bdf4b2/pkg/resources/database.go#L44, so please update the parameter description in resource definition

Reply:
Updated